sycl/source/queue.cpp

```diff
@@ -180,7 +180,9 @@ event queue::submit_impl(std::function<void(handler &)> CGH, queue SecondQueue,
 
 void queue::submit_without_event_impl(std::function<void(handler &)> CGH,
                                       const detail::code_location &CodeLoc) {
+  std::cerr << "submit_without_event_impl invoked" << std::endl;
```

Comment:
Why these print statements?

Reply:
Sorry about that, looks like I forgot to remove debug prints from that file.